kore/src/Kore/ASTVerifier/SentenceVerifier.hs

```diff
@@ -377,10 +393,26 @@ verifyClaimSentence sentence =
             $ sentenceClaimAttributes sentence
         State.modify' $ addClaim (SentenceClaim verified) attrs
   where
-    addClaim verified attrs =
-        Lens.over
-            (field @"indexedModuleClaims")
-            ((attrs, verified) :)
+    addClaim verifiedSentenceClaim@(SentenceClaim verified) attrs =
+        case fromSentenceAxiom (attrs, verified) of
+            Right (OnePathClaimPattern (OnePathRule rulePattern))
+              | rejectRulePattern rulePattern ->
+                    error "Found claim with universally-quantified variables \
```

Comment:
Please use `withSentenceClaimContext` and `koreFail` instead of `error`, so that the error message will refer to the correct line of the file.